README.md

```diff
@@ -36,6 +36,20 @@ Please consider making a tax-deductible [donation](https://numfocus.org/donate-t
   >
 </a>
 
+## Public API
+
+Our public API is documented in the [API section][] of these docs.
```

Comment:
please use roles for internal links: `func` for functions and so on.

Here, `/docs/api.md` becomes a `doc` with identifier `api`.

And again: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-pep